examples/todo/tests/test_suite.py

```diff
@@ -39,9 +42,8 @@ class DoubleToDoTest(BBTestCase):
     }
 
     def test_add(self):
-        todo_box1 = self.lab.boxes[ToDoBox.NAME][0]
-        todo_box2 = self.lab.boxes[ToDoBox.NAME][1]
-        todo_box1.add("Foo")
-        todos = todo_box1.list()
+        self.todo_box.add("Foo")
```

Comment:
hmmm.... I'm questioning the benefit of the `todo_box` property. After your rewrite the tests is less readable.